utility/doc/CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# Contribution guide
+
+- For reference, [here is the detailed version](https://github.com/pokt-network/pocket/blob/main/utility/doc/E2E_FEATURE_PATH_TEMPLATE.md#e2e-feature-implementation)
+  
+- This doc assumes you know exactly what to work on, i.e. you are assigned [an issue with details hashed out](https://github.com/pokt-network/pocket/issues/754).
```

Comment:
Couple questions come to mind:

1. How about smaller utility features?
2. Whom do I reach out to?
3. How about non-E2E features?

You don't have to answer these questions, but my worry is that a partially complete CONTRIBUTION guide (a good one is really hard to build and maintain) might raise more questions than answers.

Reply:
Thank you for the review.  I think moving this doc to be a TL;DR section in the E2E feature path template, as you suggested in another review comment, makes the most sense here.